clippy_lints/src/doc/mod.rs

```diff
@@ -420,6 +421,39 @@ declare_clippy_lint! {
     "require every line of a paragraph to be indented and marked"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    /// Detects doc comment linebreaks that use double spaces to separate lines, instead of back-slash (\).
+    ///
+    /// ### Why is this bad?
+    /// Double spaces, when used as doc comment linebreaks, can be difficult to see, and may
+    /// accidentally be removed during automatic formatting or manual refactoring. The use of a back-slash (\)
+    /// is clearer in this regard.
+    ///
+    /// ### Example
+    /// The two dots in this example represent a double space.
+    /// ```no_run
+    /// /// This command takes two numbers as inputs and..
```

Comment:
What about using one of the substitute images listed on https://en.wikipedia.org/wiki/Whitespace_character#Substitute_images instead of `.`, e.g. `·` or `␣` (not sure how well this is shown for all fonts though)

Reply:
Thanks for this list, yes, it would make more sense to use a proper substitution character. I think `·` is quite common so I'll go with that.